### PR TITLE
Add some docs, fix some bugs

### PR DESCRIPTION
**Pull request by maxrothman**
@edx/devops

### CODE REVIEW

**Comment by fredsmith** [comment]
:+1:

**Comment by maxrothman** [comment]
@edx/devops rereview

File: playbooks/roles/alton/meta/main.yml [hunk 1]
```diff
@@ -22,4 +22,6 @@
 dependencies:
   - common
   - supervisor
-  - redis
+  - role: redis
+    redis_debian_pkgs:
+      - "redis-server"
```
Comment (feanil) [inline playbooks/roles/alton/meta/main.yml]: What's up with this change?

Comment (maxrothman) [inline playbooks/roles/alton/meta/main.yml]: The redis role gets used in other places where a specific version is necessary, but the version specified in the play is too old for Will. This simply un-pins the version.

Comment (feanil) [inline playbooks/roles/alton/meta/main.yml]: Which other roles?

```
feanil@edx ~/src/configuration/playbooks/roles$ find . -name main.yml | grep meta | xargs grep redis
./alton/meta/main.yml:  - redis
./redis/meta/main.yml:# Role includes for role redis
```

I was unable to find them.

Comment (maxrothman) [inline playbooks/roles/alton/meta/main.yml]: @fredsmith, didn't you say it was on sandboxes or something?

Comment (fredsmith) [inline playbooks/roles/alton/meta/main.yml]: i checked, and it seems like nothing else uses the redis role.

Comment (feanil) [inline playbooks/roles/alton/meta/main.yml]: Let's move the pin in the redis role to whatever new version is needed to work with our current version of Will.

Comment (maxrothman) [inline playbooks/roles/alton/meta/main.yml]: Why pin at all? We're using the Ubuntu repo which won't change except for bug/security fixes.

Comment (feanil) [inline playbooks/roles/alton/meta/main.yml]: It sounded like there are versions that might break alton?  So it seems good to not accidentally get that version.

Comment (maxrothman) [inline playbooks/roles/alton/meta/main.yml]: I don't think so. The issues I ran into were with the pinned version being so old that the repo for it had disappeared.

**Comment by fredsmith** [comment]
:shipit:

File: playbooks/roles/alton/meta/main.yml (hunk 1 above)
Comment (maxrothman) [inline playbooks/roles/alton/meta/main.yml]: @feanil If there aren't any objections, I'll simply un-pin the version in the redis role.

Comment (feanil) [inline playbooks/roles/alton/meta/main.yml]: Sounds good.

**Comment by feanil** [comment]
:+1:

**Pull request merged**